x-pack/plugins/cases/docs/openapi/paths/s@{spaceid}@api@cases@{caseid}@comments@{commentid}.yaml

```diff
@@ -0,0 +1,163 @@
+post:
```

Comment:
> post:

Per previous comment, this "post" operation should be removed from this file.

Reply:
Removed in [15f9a1c](https://github.com/elastic/kibana/pull/133400/commits/15f9a1ca39e168ad061d7ae7e6fce8dbe5e9f2be).